### PR TITLE
Add old config format cause the otlp OC handler only supports OAP server 9.2.0

### PR DESCRIPTION
**Pull request by JohnDuncan5171**
Add old config format cause the otlp OC handler only supports OAP server 9.2.0

It's a supplement for [#9499] [#9521]

Most users have not been updated to OAP server 9.2.0 (such as helm charts), according to the official link to the latest document, we can't successfully deploy the otlp exporter.

### CODE REVIEW

**Comment by wu-sheng** [comment]
I don't think adding comments here would help much.

If it is a helm issue, it's better to polish docs and codrs there.

**Comment by wu-sheng** [comment]
Such as we could default versions of OAP in helm repo to 9.2.0

File: test/e2e-v2/cases/vm/prometheus-node-exporter/otel-collector-config.yaml [hunk 1]
```diff
@@ -26,6 +26,14 @@ processors:
   batch:
 
 exporters:
+  #############################################################################################
+  # If you want to use otlp exporter please ensure that your OAP version is higher than 9.2.0 #
```
Comment (kezhenxu94) [inline test/e2e-v2/cases/vm/prometheus-node-exporter/otel-collector-config.yaml:30]:
```suggestion
  # If you want to use otlp exporter please ensure that your OAP version is >= 9.2.0 #
```

**Review by kezhenxu94 — APPROVED** [review]
Could you also update the config in our demo repo? https://github.com/apache/skywalking-showcase/tree/main/deploy/platform/kubernetes

**Comment by JohnDuncan5171** [comment]
In essence, it's not a helm issue.
It's the official document issue.

We can link [Provide Linux OS monitoring](https://skywalking.apache.org/docs/main/latest/en/setup/backend/backend-vm-monitoring/) from the official UI
Then we can link [otel-collector-config.yaml](https://github.com/apache/skywalking/tree/8c683a0b67c3a20ff382489ad298828f7995c201/test/e2e-v2/cases/vm/prometheus-node-exporter/otel-collector-config.yaml) from **any version** 's this page 

This issue bothered me for 3 days. I've tried any config change/Env change to use the **otlp exporter**. until asked my friend @Ax1an to figure out if [this commit](https://github.com/apache/skywalking/commit/e1a5ee198f75c757a66671ba75cf29807045c1ff) is any version requirements

This is a very serious misleading if I don't look at the commit history

**Comment by wu-sheng** [comment]
I see. You mean the UI side links. Then I am afraid we don't have a solution to provide versionalized link for every release.

Could you follow Zhenxu suggestions to update configs in showcase repo?

**Review by wu-sheng — DISMISSED** [review]

**Comment by JohnDuncan5171** [comment]
> Could you also update the config in our demo repo? https://github.com/apache/skywalking-showcase/tree/main/deploy/platform/kubernetes

I‘m willing to support open source
But what I'm skilled at k8s/cloud native/architect 
OpenTelemetry is a new thing for me, I will do my best

**Review by kezhenxu94 — APPROVED** [review]

**Pull request merged**